app/views/feedback/_form.html.erb

```diff
@@ -1,40 +1,54 @@
       <p>
-        <%= I18n.t('general.please_fix') %> <%= @contact.errors.count %> <%= 'error'.pluralize(contact.errors.count) %> highlighted below.
+        <%= I18n.t("general.please_fix") %>
+        <%= @feedback.errors.count %>
+        <%= "error".pluralize(@feedback.errors.count) %>
+        highlighted below.
       </p>
     </div>
   <% end %>
   <div class="row mb-2">
+
     <div class="mb-3 form-group">
-      <%= f.text_field :name, autofocus: true, 
-                              class: 'form-control', 
-                              required: true %>
-      <% @contact.errors.full_messages_for(:name).each do |message| %>
+      <%= f.select :subject,
+               ["General Feedback", "Feature Request", "Bug"],
+               required: true %>
+    </div>
+    <div class="mb-3 form-group">
+      <%= f.text_field :name, autofocus: true, class: "form-control", required: true %>
+      <% @feedback.errors.full_messages_for(:name).each do |message| %>
         <div class="alert alert-danger mt-1" role="alert"><%= message %></div>
       <% end %>
     </div>
     <div class="mb-3 form-group">
-      <%= f.email_field :email, class: 'form-control', 
-                                  required: true %>
-      <% @contact.errors.full_messages_for(:email).each do |message| %>
+      <%= f.email_field :email, class: "form-control", required: true %>
+      <% @feedback.errors.full_messages_for(:email).each do |message| %>
         <div class="alert alert-danger mt-1" role="alert"><%= message %></div>
       <% end %>
     </div>
     <div class="mb-3 form-group" data-controller='counter'>
-      <%= f.text_area :message, class: 'form-control', 
-                                data: { action: "input->counter#countLarge", 
-                                        counter_target: 'input' },
-                                placeholder: t('feedback.new.message_placeholder'),
-                                required: true %>
+      <%= f.text_area :message,
+                  class: "form-control",
+                  data: {
+                    action: "input->counter#countLarge",
+                    counter_target: "input",
+                  },
+                  placeholder: t("feedback.new.message_placeholder"),
+                  required: true %>
       <div data-counter-target='output' class='small'></div>
-      <% @contact.errors.full_messages_for(:message).each do |message| %>
+      <% @feedback.errors.full_messages_for(:message).each do |message| %>
         <div class="alert alert-danger mt-1" role="alert"><%= message %></div>
       <% end %>
     </div>
     <div class="form-group">
-      <%= f.submit I18n.t('general.submit'), class: 'btn btn-primary' %>
+      <%= f.submit I18n.t("general.submit"), class: "btn btn-primary" %>
     </div>
+
+    <%= turbo_frame_tag "feedback", class: "pt-4" do %>
+      <%= link_to "Click for tips on how to submit a request", feedback_path %>
+    <% end %>
   </div>
```

Comment:
I'm wondering if this would make more sense as a Stimulus controller to toggle the visibility of the tips? Since the tips themselves are just static content, we can eliminate a request to the server.

Reply:
Good point. I was going off the issue, but now that you bring this up, I should be able to do it ever simpler with [bootstrap collapse](https://getbootstrap.com/docs/5.3/components/collapse/#how-it-works). @kasugaijin Can you think of any downside diverging from the issue requirements?

Reply:
Not at all! Providing the toggle controller is reusable in other places it makes sense as it’s a common feature. If we go that route it’s worth looking at A) stimulus - I think there is already something like this in the controllers as I believe I used toggling in the old adopter profile form. It might need a bit of a tweak. B) our theme also has toggle functionality built right in. So it would be even better to use that if it works. Eg the FAQs use a toggle feature.